.tests/testScript_offsetUnits.m

```diff
@@ -37,5 +37,5 @@
 
 shoulderror('DimVar:incompatibleUnits',"unitconversionfactor('degC','K')");
 
-%% expected to fail: this syntax is bad practice
-shoulderror("20*u.degC + 20*u.degF")
+%% this syntax is bad practice, but it works
```

Comment:
The use of the phrase "expected to fail" is admittedly very unclear (better would probably be "ideally would fail"). The usage may not be 100% consistent, but it acts as a kind of documentation for behavior that the toolbox _would_ have in the ideal case but unfortunately does not (yet) at this stage of development or because I couldn't figure out how to make it work. 

Perhaps for these cases, to get all the tests to pass but still preserve the syntax of the aspirational behavior, adding more descriptive comments to make it very clear the "this is the aspirational toolbox behavior" vs. "but this is the test that passes in the current state."

Reply:
ok thank you for explanation. I do not have a better solution yet either.